students/arun_nalla/session02/series.py

```diff
@@ -0,0 +1,78 @@
+﻿#!use/bin/env python
+def fibo(n):
+    '''Write a alternative code to get the nth value of the fibonacci series using type (list)
+    0th and 1st position should return 0 and 1 and indexing nth position should
+    return SUM of previous two numbers'''
+    if n ==0: return 0
+    elif n ==1: return 1
+    my_list = [0,1]
+    for x in range(n+1):
+        if x>=2:
```

Comment:
Instead of these 2 lines (9-10), you could have just `for x in range(2, n+1):`. You may want to look up the documentation for `range()` and get familiar with its various usages.